tests/bson/bug0592.phpt

```diff
@@ -24,14 +24,20 @@ $tests = [
 
 foreach ($tests as $json) {
     printf("Test %s\n", $json);
-    var_dump(toPHP(fromJSON($json)));
+    try {
+        $encoded = toPHP(fromJSON($json));
+        var_dump( $encoded );
+    } catch ( MongoDB\Driver\Exception\InvalidArgumentException $e ) {
+        echo "MongoDB\Driver\Exception\InvalidArgumentException: ", $e->getMessage(), "\n";
+    }
     echo "\n";
 }
 
 ?>
 ===DONE===
 <?php exit(0); ?>
 --EXPECTF--
+%a
```

Comment:
Per my earlier comment about `mongoc_log()` being useless, I think we should remove this and the `--INI--` block.